src/components/Icons/Chevron.js

```diff
@@ -38,7 +38,8 @@ ChevronIcon.defaultProps = {
 ChevronIcon.propTypes = {
   size: PropTypes.number.isRequired,
   color: PropTypes.string.isRequired,
-  direction: PropTypes.oneOf(DIRECTION)
+  direction: PropTypes.oneOf(DIRECTION),
+  onClick: PropTypes.func.isRequired
```

Comment:
let's not add click handlers to icons ... these should be wrapped in a `<button />` or `<a />` element to handle `onClicks`